zcash_keys/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this library adheres to Rust's notion of
 - `zcash_keys::keys::UnifiedIncomingViewingKey::{has_sapling, has_orchard,
   has_transparent, receiver_requirements, to_receiver_requirements}`
 - `zcash_keys::keys::ReceiverRequirements`
+- `zcash_keys::keys::UnifiedAddressRequest::{SHIELDED, ORCHARD}`
```

Comment:
This is now in the wrong place after `zcash_keys 0.8.0` was released. Rebase this PR on `main`.